src/Controls/src/Core/PlatformConfiguration/WindowsSpecific/Page.cs

```diff
@@ -49,30 +49,49 @@ public static ToolbarPlacement GetToolbarPlacement(this IPlatformElementConfigur
 
 		#region ToolbarDynamicOverflowEnabled
 
-		/// <summary>Bindable property for attached property <c>ToolbarDynamicOverflowEnabled</c>.</summary>
+		/// <summary>
+		/// Indicates whether toolbar items automatically move to the overflow menu when space is limited.
+		/// </summary>
 		public static readonly BindableProperty ToolbarDynamicOverflowEnabledProperty =
 			BindableProperty.CreateAttached("ToolbarDynamicOverflowEnabled", typeof(bool),
 				typeof(FormsElement), true);
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific/Page.xml" path="//Member[@MemberName='GetToolbarDynamicOverflowEnabled'][1]/Docs/*" />
+		/// <summary>
+		/// Gets a value that indicates whether toolbar items automatically move to the overflow menu when space is limited.
+		/// </summary>
+		/// <param name="element">A page, the VisualElement that occupies the entire screen.</param>
+		/// <returns><see langword="true"/> if toolbar items automatically move to the overflow menu when space is limited; otherwise, <see langword="false"/>.</returns>
 		public static bool GetToolbarDynamicOverflowEnabled(BindableObject element)
 		{
 			return (bool)element.GetValue(ToolbarDynamicOverflowEnabledProperty);
 		}
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific/Page.xml" path="//Member[@MemberName='SetToolbarDynamicOverflowEnabled'][1]/Docs/*" />
+		/// <summary>
+		/// Sets a value that indicates whether toolbar items automatically move to the overflow menu when space is limited.
+		/// </summary>
+		/// <param name="element">A page, the VisualElement that occupies the entire screen.</param>
+		/// <param name="value">A value that indicates whether toolbar items automatically move to the overflow menu when space is limited</param>
 		public static void SetToolbarDynamicOverflowEnabled(BindableObject element, bool value)
 		{
 			element.SetValue(ToolbarDynamicOverflowEnabledProperty, value);
 		}
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific/Page.xml" path="//Member[@MemberName='GetToolbarDynamicOverflowEnabled'][2]/Docs/*" />
+		/// <summary>
+		/// Gets a value that indicates whether toolbar items automatically move to the overflow menu when space is limited.
```

Comment:
Maybe we can expand on how this differs from `ToolbarDynamicOverflowEnabledProperty`? They seem to be identical and as a user I wouldn't know which one to use

Reply:
The `ToolbarDynamicOverflowEnabledProperty `description is:
Indicates whether toolbar items automatically move to the overflow menu when space is limited.

The usage will be for example:
```
var button = new Button() { Text = "Toggle IsDynamicOverflowEnabled" };
button.Clicked += (s, e) =>
{
     var overflowEnabled = parent.On<Windows>().GetToolbarDynamicOverflowEnabled();
     parent.On<Windows>().SetToolbarDynamicOverflowEnabled(true);
};
```